apis/apis.json

```diff
@@ -54,9 +59,13 @@
     "type": "grpc",
     "description": "Recommended Google client library to access the Google Cloud Datastore API, which accesses the schemaless NoSQL database to provide fully managed, robust, scalable storage for your application.",
     "tags": [ "Datastore" ],
+    "dependencies": {
+      "Grpc.Core": "1.4.0",
+      "Google.Api.Gax.Grpc": "2.0.0"
```

Comment:
But the point is that for stable releases we *shouldn't* be updating them.

Suppose we release GAX 2.1.0 and that becomes the default, but we then want to release Google.Cloud.Logging.V2 v2.0.1 - that *must* still use GAX 2.0.0, because Google.Cloud.Logging.V2 v2.0.0 did. When the version is bumped to 2.1.0-beta01, we'd go back to "default" for GAX and pick up GAX 2.1.0.